### DISCUSSION TITLE
custom section to a shopify theme

### DISCUSSION THREAD

**Discussion by Topdev83** [uicrooks/shopify-foundation-theme#73] [Q&A] [answered]
How to add a custom section to a Shopify theme?

**Comment by annaoutlook** [comment, marked as answer]
1. In your Shopify admin, go to "Online Store" and click on "Themes." 
2. Locate the theme you want to customize and click on "Actions" > "Edit code." 
3. In the "Sections" directory, click on "Add a new section." 
4. Create a new liquid file for your custom section (e.g.,  custom-section.liquid ) and define the structure and content of the section using HTML and Liquid. 
5. Save the file and close the code editor. 
6. Go back to the "Sections" directory and open the  schema.json  file. 
7. Add a new entry to the  blocks  array in the  schema.json  file, defining the settings for your custom section. 
8. Save the  schema.json  file and close the code editor. 
9. In your Shopify admin, go to "Online Store" > "Themes" and click on "Customize" for the theme you are working on. 
10. In the theme editor, you should now see your custom section listed under the "Sections" tab. 
11. Drag and drop the custom section to the desired location on your homepage or other pages. 
12. Customize the settings of the custom section as needed. 
13. Save and publish your changes. 
 
This process allows you to create and add custom sections to your Shopify theme, providing flexibility in designing and organizing your store's content. 
 
Please note that specific customization requirements may vary, so it's always recommended to refer to Shopify's official documentation or seek assistance from their support team for more complex customization tasks.

**Reply by Topdev83** [reply]
thanks for your answer.
